lighthouse-cli/test/smokehouse/test-definitions/forms-autocomplete.js

```diff
@@ -22,575 +21,570 @@ const config = {
-            },
-            node: {
-              nodeLabel: 'textarea',
-              snippet:
-              '<textarea type="text" id="name_cc1" name="name_cc1" autocomplete="sectio-red shipping cc-namez" placeholder="John Doe">',
-            }},
-          {
-            id: 'CCNo1',
-            name: 'CCNo1',
-            placeholder: '5555 5555 5555 5555',
-            autocomplete: {
-              property: 'cc-number',
-              attribute: 'cc-number',
-              prediction: 'HTML_TYPE_CREDIT_CARD_NUMBER',
-            },
-            node: {
-              nodeLabel: 'input',
-              snippet:
-              '<input type="text" id="CCNo1" name="CCNo1" autocomplete="cc-number" placeholder="5555 5555 5555 5555">',
-            }},
-          {
-            id: 'CCExpiresMonth1',
-            name: 'CCExpiresMonth1',
-            autocomplete: {
-              property: 'cc-exp-month',
-              attribute: 'cc-exp-month',
-              prediction: 'HTML_TYPE_CREDIT_CARD_EXP_MONTH',
-            },
-            node: {
-              nodeLabel: 'MM\n01\n02\n03\n04\n05\n06\n07\n08\n09\n10\n11\n12',
-              snippet:
-              '<select id="CCExpiresMonth1" name="CCExpiresMonth1" autocomplete="cc-exp-month">'},
+        {
+          parentFormIndex: 0,
+          labelIndicies: [
+            1,
+          ],
+          id: 'CCNo1',
+          name: 'CCNo1',
+          type: 'text',
+          placeholder: '5555 5555 5555 5555',
+          autocomplete: {
+            property: 'cc-number',
+            attribute: 'cc-number',
+            prediction: null,
           },
-          {
-            id: 'CCExpiresYear1',
-            name: '',
-            autocomplete: {
-              property: 'section-red billing cc-exp-year',
-              attribute: 'section-red billing cc-exp-year',
-              prediction: 'HTML_TYPE_CREDIT_CARD_EXP_YEAR',
-            },
-            node: {
-              nodeLabel: 'YY\n2019\n2020\n2021\n2022\n2023\n2024\n2025\n2026\n2027\n2028\n2029',
-              snippet:
-              '<select id="CCExpiresYear1" autocomplete="section-red billing cc-exp-year">',
-            }},
-          {
-            id: 'cvc1',
-            name: 'cvc1',
-            placeholder: '555',
-            autocomplete: {
-              property: 'cc-csc',
-              attribute: 'cc-csc',
-              prediction: 'HTML_TYPE_CREDIT_CARD_VERIFICATION_CODE',
-            },
-            node: {
-              nodeLabel: 'input',
-              snippet: '<input id="cvc1" name="cvc1" autocomplete="cc-csc" placeholder="555">'},
+        },
+        {
+          parentFormIndex: 0,
+          labelIndicies: [
+            2,
+          ],
+          id: 'CCExpiresMonth1',
+          name: 'CCExpiresMonth1',
+          type: 'select-one',
+          autocomplete: {
+            property: 'cc-exp-month',
+            attribute: 'cc-exp-month',
+            prediction: null,
```

Comment:
all predictions are  now null. lets add a comment.. somewhere in this file... to clarify that they'd be populated if the flag were used.

Reply:
There's a comment below that mentions the flag

Reply:
i saw that. but it's for the section you commented out. :)
i don't think the comment on line 467 is gonna help give context to whats happening on line 70.. 
so i was suggesting an additional comment.